### ISSUE TITLE
T-configuration issue

### ISSUE THREAD

**Issue by DarthGoogle** [mezzodrinker/spas#15]
Ive set up this script with a T-configuration. It uses a small ship solars on a station via large-to-small rotors connection.

Ive set script's solar settings according to recommended settings for small ships.

Set 2 groups of rotors (though horizontal one only have one rotor so far)
Set 1 group of solar panels to which i assigned all solars that i named uniquely just in case.

The problem is that one half of the whole array always turns on an angle with the other. First gets almost perfect alignment, but the second gets an angle ranging from 90 to 45 degree approx.

In other words, half of array is de-xynced with another.

Ive tried to set up an inverted rotor but to no avail. Tried [inv]%RotorName% both with and without space.

Does not work. Both rotors, inverted or not, are assigned to the same group.

**Comment by mezzodrinker** [comment]
Which version of the script are you using? Have you set the braking torque of the rotors?

**Comment by DarthGoogle** [comment]
Im using latest version from the Steam Workshop. Braking force is set to the max on both Large-to-Small and small rotors, which differs.

**Comment by mezzodrinker** [comment]
What do you mean by "large-to-small and small rotors"? Do you have a screenshot of your setup?

**Comment by DarthGoogle** [comment]
To be more specific:

Horizontal rotor is a Large-To-Small
Both vertical rotors are Small-To-Small

Braking force is determined by the rotor base which means i have different limits for horizontal nad vertical axis, but this should not affect the panel behavior as far as i understand.

**Comment by mezzodrinker** [comment]
Looks like it could work. Though I have no idea why it does not. What happens if you replace the "large-to-small" (and thus the small rotors) with a regular rotor/advanced rotor and adjust the script to use the newly placed rotors?

**Comment by DarthGoogle** [comment]
Im going to experiment with different setups in creative tomorrow using few setups: all-large, all-small, X large-to-small and Y large-to-small. Will post a details here once i do.

**Comment by mezzodrinker** [comment]
That'd be great, thank you.

**Comment by DarthGoogle** [comment]
Tested with small and large rotors so far.

One single and one T config for each, four total.

It aligns, though again, far from perfection: but nothing too extreme as before when half solars got aligned 90 degrees towards the sun, but misaligment is still there.

Its almost like it gets worse the more solar panels are used for optimization.

**Comment by mezzodrinker** [comment]
Do the solar panels now align "in sync", meaning that both sides face the same direction?

**Comment by DarthGoogle** [comment]
Sometimes, but the sync is far from perfect.

Both sides face the sam direction though the output is suboptimal. Angle between halfs changes over time, sometimes increasing or decreasing, but they dont line up or end up being close to parrallel

http://images.akamai.steamusercontent.com/ugc/278470593334091027/314DAF89FA44852F3796ED2853DBBF50FD2446E9/

**Comment by mezzodrinker** [comment]
That's weird. I can't reproduce it on my side.

**Comment by DarthGoogle** [comment]
I could upload the world. But it sounds like im actually making a mistake in setting up the script Oo

**Comment by mezzodrinker** [comment]
Would you mind giving me a step-by-step guide what you did to set up your solar towers?

**Comment by DarthGoogle** [comment]
1) Put a new station or small ship on the ground.
2) Set a battery, timer block, prog.block, horizontal rotor. 
3) Add few more blocks on top of horizontal rotor. Place vertical rotor, then rotate and place one on the other side to make a T-cross.
4) Change all ownership to 'me' just in case

Then i change the scrip setting etc. etc. If i use small solar panels, i set up recommended values for those since script itself defaults to large ones, both for average output and powersaver feature.

**Comment by mezzodrinker** [comment]
Well, the part where you put "etc. etc." is actually the important part.

**Comment by DarthGoogle** [comment]
For T-config it is:

Horiontal rotor is put to a group named Rotors (X)
Vertical rotrs are Rotors (Y)

GroupName string is (true,true)

Solar panels are in a group named Solar Panels (SPAS)

String which asks if the script shouls use a group name is (true)

Solars are named: Solar Panel (SPAS) with a number increasing on each to give a unique name

Rotors are the same being Rotor X(or Y) numbered for same reason.

One of the Y Rotors is actually named [inv] Rotor 2 Y with or without space.

**Comment by mezzodrinker** [comment]
Did you close this issue on purpose?

**Comment by DarthGoogle** [comment]
Nope, sorry, misclick

**Comment by DarthGoogle** [comment]
Do you need more specific information or tests?

**Comment by mezzodrinker** [comment]
Most likely linked to #16.
